app/assets/javascripts/discourse/app/models/post.js

```diff
@@ -355,21 +355,18 @@ const Post = RestModel.extend({
     }
   },
 
-  updateLikeCount(count) {
+  updateLikeCount(count, userId, eventType) {
+    let ownLike = User.current()?.id === userId && eventType === "liked";
     let current_actions_summary = this.get("actions_summary");
     let likeActionID = Site.current().post_action_types.find(
       (a) => a.name_key === "like"
     ).id;
+    let newActionObject = { id: likeActionID, count, acted: ownLike };
```

Comment:
```suggestion
    const newActionObject = { id: likeActionID, count, acted: ownLike };
```

Doesn't look like this is being modified so a constant should suffice.